kommunicateui/src/main/res/values/strings.xml

```diff
@@ -227,4 +227,26 @@
     <string name="km_active_bots_text">Active bot:</string>
     <string name="km_default_assigned_bot_name">Support Bot</string>
     <string name="km_take_over_from_bot_button_text">Take over from bot</string>
+    <string name="resolve_info_text1">later</string>
+    <string name="resolve_info_text2">la</string>
+    <string name="resolve_info_text3">te</string>
+    <string name="rating_status_text">r</string>
+    <string name="conversation_rated_info">You rated this conversation</string>
+    <string name="have_other_queries">Have other queries?</string>
+    <string name="resolved_info_text">This conversation has been marked as resolved.</string>
+    <string name="restart_conversation_text">Restart conversation</string>
+    <string name="rating_text">You rated the conversation</string>
+    <string name="rating_good">Good</string>
+    <string name="add_a_comment">Add a comment..</string>
+    <string name="submit_your_rating">Submit your rating</string>
+    <string name="rating_average">Average</string>
+    <string name="rate_the_conversation">Rate the conversation</string>
+    <string name="rating_poor">Poor</string>
+    <string name="feedback_update_failed">"Error occurred. Feedback update failed."</string>
+    <string name="mandatory_fields">"Mandatory fields required.."</string>
+    <string name="press_back_exit">"Press Back again to Exit."</string>
+    <string name="enter_user_id">"Enter UserId."</string>
```

Comment:
@shubhamtewari check if . is required, check for all strings added. we are not adding . in other existing strings

Reply:
Went through google design guidelines and added and removed full stops accordingly, as per the examples seen.